src/app/main.rs

```diff
@@ -204,7 +205,7 @@ where
         );
         info!(
             "serving Prometheus metrics on {:?}",
```

Comment:
Nit: perhaps this should be
```suggestion
            "serving admin endpoint metrics on {:?}",
```
or something now?